app/helpers/application_helper.rb

```diff
@@ -376,11 +376,7 @@ def pluralize_without_count(count, noun, text=nil)
   end
 
   def get_page_name
```

Comment:
Nice -- not only is it good to have this pulled into a service, but pulling it into MetricsService clarifies that this is a page name that's only used for analytics